### ISSUE TITLE
Unet训练验证时，MIOU为0，precission为0，recall为0，Kappa为0。

### ISSUE THREAD

**Issue by SweepingMonkWWW** [PaddlePaddle/PaddleSeg#2755]
当使用unet模型训练验证时，MIOU为0，precission为0，recall为0，Kappa为0。配置文件相同，其他模型均正常，请问是什么原因？
![f677549e4ca3114859a299743d636ab](https://user-images.githubusercontent.com/118884219/203498731-c6567474-b32f-4c86-98a0-53353464caa6.png)

**Comment by SweepingMonkWWW** [comment]
保证数据集一致，配置文件一致，利用其他模型训练时则正常，unet训练就不行。下图为其他模型评估时的结果。
![5826fd68be135e5dd799536cbb91c9b](https://user-images.githubusercontent.com/118884219/203499194-2c6ab895-6fa2-49a9-bbdf-8dfc18f7e095.png)

**Comment by SweepingMonkWWW** [comment]
相关配置文件如下
_base_: '../_base_/cityscapes.yml'

batch_size: 2
iters: 1000

model:
  type: UNetPlusPlus
  in_channels: 3
  num_classes: 2
  use_deconv: False
  align_corners: False
  pretrained: Null
  is_ds: True


batch_size: 2
iters: 80000

train_dataset:
  type: Dataset
  dataset_root: data/positive_data
  train_path: data/positive_data/train_list.txt
  num_classes: 2
  mode: train
  transforms:
    - type: ResizeStepScaling
      min_scale_factor: 0.5
      max_scale_factor: 2.0
      scale_step_size: 0.25
    - type: RandomPaddingCrop
      crop_size: [512, 512]
    - type: RandomHorizontalFlip
    - type: RandomDistort
      brightness_range: 0.5
      contrast_range: 0.5
      saturation_range: 0.5
    - type: Normalize

val_dataset:
  type: Dataset
  dataset_root: data/positive_data
  val_path: data/positive_data/val_list.txt
  num_classes: 2
  mode: val
  transforms:
    - type: Normalize


optimizer:
  type: sgd
  momentum: 0.9
  weight_decay: 4.0e-5

lr_scheduler:
  type: PolynomialDecay
  learning_rate: 0.01
  end_lr: 0
  power: 0.9

loss:
  types:
    - type: CrossEntropyLoss
  coef: [1]

**Comment by wuyefeilin** [comment]
只训练了1000iter，太短了，模型还不能收敛，训练长一些
